src/cc2olx/enums.py

```diff
@@ -14,3 +15,23 @@ class CommonCartridgeResourceType(str, Enum):
     QTI_ASSESSMENT = r"^imsqti_xmlv\d+p\d+/imscc_xmlv\d+p\d+/assessment$"
     DISCUSSION_TOPIC = r"^imsdt_xmlv\d+p\d+$"
     ASSIGNMENT = r"^assignment_xmlv\d+p\d+$"
+
+
+class SupportedCustomBlockContentType(str, Enum):
```

Comment:
FWIW, if you're doing this instead of using `enum.StrEnum` because you're concerned about backwards compatibility with Python 3.8, I think you can move cc2olx's requirements up to 3.11. Pretty much all the other repos in openedx have been upgraded to that, and 3.8 is past end of life anyway.